modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -82,6 +96,18 @@ image::rlp-conditions.png[]
 +
 If a RAML spec is attached to your API, click *Preview resource matching* to verify which resources your filters affect.
 
+[[register-deploy]]
+== Register and Deploy
+After applying the policy, the following steps are required in order for the policy to take effect:
+
+. Register a client application to access the API with the newly applied policy:
+.. xref:exchange::to-request-access.adoc[Request access from Exchange.]
+.. xref:manage-client-apps-latest-task.adoc[Approve access.]
+.. xref:access-client-app-id-task.adoc[Obtain the client credentials of the registered client application.]
+. Deploy the application:
+.. xref:runtime-manager::deploying-to-cloudhub.adoc[Deploy to CloudHub.]
+. Test using a tool such as Advanced REST Client, making sure to populate the request's `client_id` and `client_secret` header fields with the client credentials obtained earlier.
```

Comment:
```suggestion
. Test using a tool such as Advanced REST Client, making sure to populate the request's `client_id` and `client_secret` header fields with the client credentials obtained earlier.

```
Test what using the ARC? The "making sure" sentence should be in step explained in the next sentence as something like
When you test the..., ensure that you provide the request's `client_id` and `client_secret` header fields with the client credentials obtained previously in the procedure? Or where?

Reply:
Good points. I added what is to be tested, and when the credentials were obtained.